build.xml

```diff
@@ -251,41 +251,45 @@ limitations under the License.
         <run script="${build.test}/testXliffMerge.js" executable="${nodeunit}/bin/nodeunit" dir="${build.test}" args="" />
     </target>
 
+    <target name="test.xliffsplit" description="Run only the xliff merge tests">
```

Comment:
Maybe next time, please do the whitespace fixing in a separate PR, as it hides this change.

Reply:
Yep, I'll seperate it next time